cli/cmd/env/set.go

```diff
@@ -56,6 +61,5 @@ func SetCmd(ch *cmdutil.Helper) *cobra.Command {
 	setCmd.Flags().StringVar(&projectName, "project", "", "Cloud project name (will attempt to infer from Git remote if not provided)")
 	setCmd.Flags().StringVar(&projectPath, "path", ".", "Project directory")
 	setCmd.Flags().StringVar(&environment, "environment", "", "Optional environment to resolve for (options: dev, prod)")
```

Comment:
Unrelated, but I guess this should be:

```suggestion
	setCmd.Flags().StringVar(&environment, "environment", "", "Optional environment to set for (options: dev, prod)")
```